src/pages/srp/components/BookMsg.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="modal is-active">
+    <div class="modal-background"></div>
+    <div class="modal-content" style="width:auto">
+      <!-- Any other Bulma elements you want -->
+      <article class="message is-success" v-if="resMsg.IsSuccess">
+        <div class="message-header">
+          <p>Success</p>
+          <button
+            v-on:click="onCancel"
+            class="delete"
+            aria-label="delete"
+          ></button>
+        </div>
+        <div class="message-body">
+          <p>Thank You for Choosing Us!!</p>
+          <p>Our Team will soon get back to you!!</p>
+        </div>
+      </article>
+      <article class="message is-warning" v-if="!resMsg.IsSuccess">
+        <div class="message-header">
+          <p>Warning</p>
+          <button
+            v-on:click="onCancel"
+            class="delete"
+            aria-label="delete"
+          ></button>
+        </div>
+        <div class="message-body">
+            <p>Sorry for the inconvenience!! </p>
+          <p>It seems Spot is already filled...</p>
```

Comment:
we can replace this with resMsg.DisplayMsg

Reply:
done